.github/workflows/publish.yaml

```diff
@@ -0,0 +1,26 @@
+name: Publish
+on: [push]
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout 🛎️
+        uses: actions/checkout@v2 # If you're using actions/checkout@v2 you must set persist-credentials to false in most cases for the deployment to work correctly.
+      - name: setup node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Install and Build 🔧 # convert the yaml schema to json version and docs, and publish on gh-pages
+        run: |
+          npm install js-yaml
+          # pip install json-schema-for-humans
+          mkdir schemas/input/latest
+          js-yaml schemas/input/csl-data.yaml > schemas/input/latest/csl-data.json # currently leave off, as no yaml file
+          # TODO generate-schema-doc -–deprecated-from-description schemas/input/csl-data.yaml build/csl-data/index.html
+
+      - name: Deploy 🚀
```

Comment:
Just realized you probably want to make this step conditional... you don't want to deploy pull requests or forks or perhaps non-master branches.

Example at https://github.com/manubot/rootstock/blob/ce827c7789163dca33dfa3bd2df7aa828e57485d/.github/workflows/manubot.yaml#L57